src/django_upgrade/fixers/testcase_methods.py

```diff
@@ -0,0 +1,43 @@
+"""
```

Comment:
I think there's a better name that `testcase_methods` - the pre-existing `assert_form_error` covers *other* testcase methods